src/state/AttachState.ts

```diff
@@ -84,13 +84,13 @@ export class AttachState implements IState{
         }
 
         await this.attachContainerLogs(CONSENSUS_NODE_LABEL);
-        await this.attachContainerLogs(MIRROR_NODE_LABEL)
+        await this.attachContainerLogs(MIRROR_NODE_LABEL);
         await this.attachContainerLogs(RELAY_LABEL);
 
         let i = 0;
-        while (i++ < Number.MAX_VALUE) {
+        while (i++ < this.loopIterations()) {
```

Comment:
Оh ... no no. Now I understand why the Attached State is so slow. 
let's talk offline.

Reply:
If i put while(true) here I cannot control the number of iterations and test it, because the timeout is 20 seconds and the tests fail. If you have any ideas on how to test, Im open to suggestions, cause I know currently this is not the best solution

Reply:
Normally ... the best and most accurate update loops can be found in the gaming industry. For obvious reasons.
[Example](https://timetocode.tumblr.com/post/71512510386/an-accurate-nodejs-game-loop-inbetween-settimeout)

But I guess the current approach is not the worst thing ...